serverless/verify-backup/src/notify-slack/index.js

```diff
@@ -0,0 +1,25 @@
+const { IncomingWebhook } = require('@slack/webhook')
```

Comment:
Can we use sentry? Create another project / use the serverless one.

Reply:
this is actually not being used?

Reply:
I didn't write a script to deploy this, this lives as a cloud function in gcp which is triggered on failure to restore dump. Okay will include sentry in it.

Reply:
haha I meant to use sentry instead of the slack webhook

Reply:
because sentry already sends results to slack